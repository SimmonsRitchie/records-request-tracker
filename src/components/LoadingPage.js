import React from 'react';

const LoadingPage = () => (
    <div className="loader">
        <img src="/images/loader.gif" />
    </div>
);

export default LoadingPage;