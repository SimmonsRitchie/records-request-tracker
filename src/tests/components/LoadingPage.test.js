import React from 'react';
import LoadingPage from '../../components/LoadingPage'
import { shallow } from 'enzyme'

// Using beforeEach, we can save ourselves from re-writing spies
// and the wrapper.
let wrapper;

beforeEach(() => {
    wrapper = shallow(<LoadingPage 
        />)
})

test('Should render LoadingPage correctly', () => {
    expect(wrapper).toMatchSnapshot()
});