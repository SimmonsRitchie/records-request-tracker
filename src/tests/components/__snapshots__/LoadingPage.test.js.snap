// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render LoadingPage correctly 1`] = `
<div
  className="loader"
>
  <img
    src="/images/loader.gif"
  />
</div>
`;
